### PR TITLE
Mark orders

### PR DESCRIPTION
**Pull request by JasjotBenipal**
This is for store owner to view orders and confirm orders using button. I don't have any connection with database and it is only local hardcoded values

### CODE REVIEW

File: app/src/main/java/com/example/b07group7project/StoreOwnerActivity.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/StoreOwnerActivity.java:33]: redo this part so it works async. Basically instead of returning a List<Order>, you should provide a function that will be called with the list of orders

Comment (aarondme) [reply app/src/main/java/com/example/b07group7project/StoreOwnerActivity.java:33]: https://github.com/sjingoi/B07Group7Project/pull/17/commits shows an example refactoring to get that to work

**Review by aarondme — CHANGES_REQUESTED** [review]
The above should be done before we merge

**Comment by aarondme** [comment]
How should I test this?

**Review by aarondme — APPROVED** [review]
Approving to prevent a blocker. I'd like that one change, or a reason why it can't be changed. Second approver, please run the code and test that the display is decent.

**Review by HariBala246 — APPROVED** [review]

**Review by Zain4Rizvi — APPROVED** [review]
I think its good. Defines Interface and calls from database so should be ok. Not sure if Order having order_complete field should be there but seems ok.

**Review by sjingoi — APPROVED** [review]
Very Basic features done, a list of products should be provided in a new branch

**Pull request merged**
